com.valvesoftware.Steam.Utility.protontricks.yml

```diff
@@ -0,0 +1,108 @@
+---
+id: com.valvesoftware.Steam.Utility.protontricks
+branch: stable
+runtime: com.valvesoftware.Steam
+runtime-version: stable
+sdk: org.freedesktop.Sdk//19.08
+build-extension: true
+appstream-compose: false
+build-options:
+  prefix: /app/utils/protontricks
+  prepend-path: /app/utils/protontricks/bin
+  prepend-pkg-config-path: /app/utils/protontricks/lib/pkgconfig
+  prepend-ld-library-path: /app/utils/protontricks/lib
+  strip: true
+cleanup:
+  - "*.a"
+  - "*.la"
+  - /include
+  - /lib/pkgconfig
+  - /share/aclocal
+  - /share/man
+modules:
+
+  - name: protontricks
+    buildsystem: simple
+    build-commands:
+      - python3 setup.py install --prefix=${FLATPAK_DEST} --root=/
+    post-install:
+      - install -Dm644 -t ${FLATPAK_DEST}/share/metainfo ${FLATPAK_ID}.metainfo.xml
+      - appstream-compose --basename=${FLATPAK_ID} --prefix=${FLATPAK_DEST} --origin=flatpak ${FLATPAK_ID}
+    sources:
+      - type: git
+        url: "https://github.com/Matoking/protontricks.git"
+        tag: "1.3"
+        commit: 5bfe33ea8bbe0c72b7c2175dcb4c7bd4f9fc9434
+      - type: file
+        path: com.valvesoftware.Steam.Utility.protontricks.metainfo.xml
```

Comment:
Since you are the developer, consider moving appdata upstream. It's not a flatpak-specific thing, other may benefit from it as well.